packages/client-twitter/src/interactions.ts

```diff
@@ -226,17 +224,6 @@ export class TwitterInteractionClient {
         };
         const currentPost = formatTweet(tweet);
 
-        let homeTimeline: Tweet[] = [];
-        // read the file if it exists
-
-        const cachedTimeline = await this.client.getCachedTimeline();
```

Comment:
This is incorrectly named "timeline" - it was really just a list of the bot's previous posts

Reply:
Meh, I’ve heard it called timeline. Profile timeline might be more apt and precise

Reply:
yeah, we can revisit this in a future pr. didn't actually change the underlying function

Reply:
lgtm, if we need to have a list of previous posts for some analysis, we can quickly re-introduce it. For the PR its good